src/main/java/mate/model/Car.java

```diff
@@ -9,6 +9,27 @@ public class Car {
     private Manufacturer manufacturer;
     private List<Driver> drivers;
 
+    public Car() {
+    }
+
+    public Car(String model, Manufacturer manufacturer) {
+        this.model = model;
+        this.manufacturer = manufacturer;
+    }
+
+    public Car(String model, Manufacturer manufacturer, List<Driver> drivers) {
+        this.model = model;
```

Comment:
Please see the example in common mistakes for this suggestion: "Remember about this() when you create a lot of constructors" .